pkg/common-controller/snapshot_controller.go

```diff
@@ -546,6 +622,7 @@ func (ctrl *csiSnapshotCommonController) getDynamicallyProvisionedContentFromSto
 	if content.Spec.Source.VolumeHandle == nil {
```

Comment:
Should we record a snapshotStatusTypeInvalidRequest when it returns at line 619?

```
	if content == nil {
		// no matching content with the desired name has been found in cache
		return nil, nil
	}
```